Tails specific: Tor is started after HTP via NetworkManager hooks.

--- chroot.orig/etc/init.d/tor	2012-09-24 10:05:13.329056459 +0200
+++ chroot/etc/init.d/tor	2012-09-24 10:03:30.638108333 +0200
@@ -9,2 +9,2 @@
-# Default-Start:     2 3 4 5
+# Default-Start:
-# Default-Stop:      0 1 6
+# Default-Stop:      0 1 6 2 3 4 5
